docs/core/idioms/logging.md

```diff
@@ -0,0 +1,86 @@
+# Logging configuration and usage
+
+Prefect natively ships with a default logger configured to handle the management of logs. More information on logging in Prefect can be found in the [logging concept document](/core/concepts/logging.html).
+
+### Configure the logger
+
+The default Prefect logger can be configured by adjusting the settings found in [Prefect's config](/core/concepts/configuration.html).
+
+```toml
+[logging]
```

Comment:
We just introduced a new config for logging `log_attributes` that we should mention here; it can be used to include context info in your log formats